tests/Api.IntegrationTests/Endpoints/App/AppTests.cs

```diff
@@ -31,11 +29,8 @@ public AppTests(PasswordlessApiFactory apiFactory)
     [InlineData("1")]
     public async Task I_cannot_create_an_account_with_an_invalid_name(string name)
     {
-        // Arrange
-        var request = AppCreateGenerator.Generate();
-
         // Act
-        var response = await _client.PostAsJsonAsync($"/admin/apps/{name}/create", request);
+        using var response = await CreateApplication(_client, name);
```

Comment:
Maybe convert it to an extension method on `HttpClient`? Similar to `AddManagementKey()`.